Content: Remnant culture: Paradise 1 - Historic Spaceport

Zitchas opened the pull request:
**Content (Artwork / Missions / Jobs)**

## Summary
This adds a "culture mission" wherein Taely expresses curiosity about Vail, and asks if you can take her there in exchange for a tour of a historic Remnant spaceport.

## Details
Lore Impacts:
- Remnant don't have cars, due to historic shortage of mass industrial capacity.
- Hover platforms that the camels tow are actually spaceships stripped down to nothing but a flat platform with a minimal set of repulsors.
- One of the initial planetary threats on Caelian was a predator that hunted in swarms and acted as a team to triangulate prey based on sound. Said predator could climb very well but couldn't navigate overhangs.
- They rely on trains for high-traffic on-planet routes, with high-speed maglev style trains.
- Incidental low-speed travel tends to happen by camel train. People exploring the planetary surface are often gone on week or month long expeditions.
- A segment of the Remnant population lives nomadic lifestyles exploring their worlds.
- Windturbines, barrels, wood, and other "archaic" technologies were instrumental in early colonization efforts and still in use today.
- Taely considers having the industrial base to have vast multitudes of different cars and vehicles and all the support industries that provide parts and whatnot for them to be a "luxury" that she envies.
- Some Caelian vegetation is similar enough to Earth that the camels can actually eat it. Not very well, only sustains them at a "marginally better than starving" level, but developing new bacterial strains to aid their digestion improved the situation enough that camels can now survive comfortably on native vegetation.

## Save File
This save file can be used to play through the new mission content:
[Zitchas Windrider.txt](https://github.com/endless-sky/endless-sky/files/8371558/Zitchas.Windrider.txt)

## Artwork Checklist
 - [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [X] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: [link to the PR](https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/19)
 - [X] Assets are uploaded to a [zip on google drive.](https://drive.google.com/file/d/1kzOoUuWRyZrD4-823e4cHmF2ZOD9EIKj/view?usp=sharing)
 - [X] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: [It can be found here.](https://github.com/endless-sky/endless-sky-high-dpi/pull/227)

Zitchas commented:
I'll just say this right up front: Yes, this has a conversation loop. Basically every RPG out there uses them for situations like this where it is a side mission and the player is allowed to just ask questions about a variety of things. If they are good enough for the top AAA industry RPG studios, I don't see any reason why we can't use them when appropriate.

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+			`As you approach your navigation system easily picks up the beacons designating the approach path, and before too long you connect to the automated approach control that shunts you into the appropriate route down to a landing pad on the outskirts of the main spaceport.`
+				goto vaillanding
+			label science
+			`As you approach Vail your sensors light up with energy traces that it quickly identifies as part of the weather control network. From what you recall, many areas of the planet have been adjusted to have greater than usual snowfall. As you approach closer your navigation system easily picks up the beacons designating the approach path, and before too long you connect to the automated approach  ...
```
```suggestion
			`As you approach Vail, your sensors detect parts of the weather control network tasked with increasing the amount of snowfall on the planet. Your navigation system picks up the beacons designating the approach path, and before too long you are automatically guided down to a landing pad on the outskirts of the main spaceport.`
```

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+			label vaillanding
+			`	Looking outside the ship, you can see a multitude of tourist-oriented facilities overwhelming the normal spaceport functions. You can pick out a small sign off to the side that indicates professional commodity and banking facilities, but otherwise the area is dominated by businesses offering mountain touring, sports classes, and vehicle rentals. Next to you, Taely alternates between peering  ...
+			choice
+				`	"Looks good to me."`
+				`	"Alright, should be good."`
```
"Rather than offer you the illusion of free choice, I will take the liberty of choo-sing for you."

Zitchas replied:
That is one thing I have learned from Mass Effect: Illusion of choice actually can be important: The vast majority of "choices" in Mass Effect have zero impact on the course of events of the game. They do, however, let the player shape their character. Am I the rebellious, crass, and aggressive commander that does stuff in the most gratingly hostile way I can? Am I the easy-going and/or professional soldier that accepts things the way they are and gets along with people? Some mix somewhere between the two? 

Exact same course of events, different story.

Anarchist2 replied:
This isn't the illusion of choice, though. You are offering the exact same choice twice. Like at this point, you might as well chuck out the choice node and just jump straight to the boarding.

Zitchas replied:
Fair, but me, as a player, really wants to be able to say something there.

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+			choice
+				`	"Looks good to me."`
+				`	"Alright, should be good."`
+			branch "havehoverboarded"
```
```suggestion
			branch havehoverboarded
```

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+				`	"Alright, should be good."`
+			branch "havehoverboarded"
+				has "remnant: hoverboarding"
+			`	The resort in question is a more exclusive facility, and thus is located away from the main train lines. You rent a hovercar from one of the rental companies and make the short trek out to the resort. The rest of the day is spent out on the slopes, where both you and Taely eventually develop sufficient skill to graduate from the easy runs up to the medium difficulty. You both fall repeatedly ...
```
```suggestion
			`	The resort in question is a more exclusive facility, and is located away from the main train lines. You rent a hovercar from one of the rental companies and make the short trek out to the resort. The rest of the day is spent out on the slopes, where both you and Taely eventually develop sufficient skill to graduate from the easy runs up to the medium difficulty. You both fall repeatedly, but Taely's good humor is contagious as she takes the inevitable facefuls of snow in stride.`
```

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+			`	The resort in question is a more exclusive facility, and thus is located away from the main train lines. You rent a hovercar from one of the rental companies and make the short trek out to the resort. The rest of the day is spent out on the slopes, where both you and Taely eventually develop sufficient skill to graduate from the easy runs up to the medium difficulty. You both fall repeatedly ...
+				goto vailheadhome
+			label havehoverboarded
+			`	The resort in question is a more exclusive facility, and thus is located away from the main train lines. You rent a hovercar from one of the rental companies and make the short trek out to the resort. The rest of the day is spent out on the slopes, where you teach Taely the basics of hoverboarding. By the end of the day Taely has impressed you with her quick learning and you finish off with  ...
```
```suggestion
			`	The resort in question is a more exclusive facility, and is located away from the main train lines. You rent a hovercar from one of the rental companies and make the short trek out to the resort. The rest of the day is spent out on the slopes, where you teach Taely the basics of hoverboarding. By the end of the day Taely has impressed you with her quick learning and you finish off with a few runs down some of the more challenging slopes. Taely spills a few times, but she takes her failures in good humor, and never seems to make the same mistake twice.`
```

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+			`	Heading back to the spaceport after a day of hoverboarding, Taely surprises you by pulling the hood open on the hovercar and spending almost an hour going over the internal systems before closing it up with a nod. "Well, that was interesting. The hovercar, I mean." She gestures at it with her commlink. "So many different manufacturers were involved in making this vehicle. Sometimes I wonder  ...
+	on complete
+		conversation
+			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with gulls and pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a larg ...
```
```suggestion
			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with Gulls and Pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a large platform.`
```

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+	on complete
+		conversation
+			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with gulls and pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a larg ...
+			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier." She says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature and will not cause damage, but they can still hurt if you run into them."`
```
```suggestion
			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier," she says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature, but they can still hurt if you run into them."`
```

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+		conversation
+			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with gulls and pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a larg ...
+			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier." She says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature and will not cause damage, but they can still hurt if you run into them."`
+			`	Moments later you see why the barriers are in place, as a large train whistles into the station at high speed and slows quickly to a stop in front of you. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering or exiting the train. Within moments the doors close and the train starts moving ...
```
```suggestion
			`	Moments later, you see why the barriers are in place as a large train whistles into the station at high speed before stopping. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering and exiting the train. Within moments the doors close and the train starts moving again as you are pressed back into your seat. You see a few lights flicker past outside the window, but soon there is nothing outside but a monotonous dark blur.`
```

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with gulls and pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a larg ...
+			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier." She says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature and will not cause damage, but they can still hurt if you run into them."`
+			`	Moments later you see why the barriers are in place, as a large train whistles into the station at high speed and slows quickly to a stop in front of you. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering or exiting the train. Within moments the doors close and the train starts moving ...
+			`	Taely notices your gaze. "Sorry, but there will not be much to see out there. The entire train line is quite far underground. Far enough to simply pass under terrain obstacles like rivers, valleys, or chasms." She nods to herself in satisfaction. "This will not be a long trip: We will only spend a few minutes at top speed before we start decelerating to our destination. I would recommend buc ...
```
```suggestion
			`	Taely notices your gaze. "Sorry, but there will not be much to see out there. The entire train line is quite far underground. Far enough to simply pass under terrain obstacles like rivers, valleys, or chasms." She nods to herself in satisfaction. "This will not be a long trip: we will only spend a few minutes at top speed before we start decelerating near our destination. I would recommend buckling into the harness."`
```

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+	stopover "Vail"
+	to offer
+		has "event: remnant: cognizance calm"
+		or
```
It's not, actually. The random value must be greater than 30 to trigger normally. With a defer, the random value needs to be greater than 60. So that should be something close to half as likely to trigger.

Anarchist2 commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+	to offer
+		has "event: remnant: cognizance calm"
+		or
+			and
+				random > 30
+				not "Remnant: Visit to Paradise 1: defer"
+			and
+				random > 60
+				has "Remnant: Visit to Paradise 1: defer"
```
Keep in mind that "Mission: defer" conditions aren't generated automatically.
```suggestion
			and
				random < 70
				not "Remnant: Visit to Paradise 1: defer"
			and
				random < 40
				has "Remnant: Visit to Paradise 1: defer"
```

Zitchas replied:
They're not? We have to manually create those when we use a defer?

Anarchist2 replied:
Yes.

Zitchas replied:
Alright, thanks for pointing that out. I'll add in an "on defer" clause to add in that statement.

Zitchas commented on data/remnant/remnant 2 cognizance.txt:
```diff
@@ -1618,3 +1618,13 @@ mission "Remnant: Cognizance 38"
+	to offer
+		has "Remnant: Cognizance 38: done"
+	on offer
+		event "remnant: cognizance calm" 5
```
The question, on my part, is this: "How long can I expect the player to fly around in circles doing nothing (or just doing generic jobs) after the end of the storyline before they're just going to head off for elsewhere?

This 5 day gap is basically just supposed to be a moment for people to catch their breath before more story stuff starts triggering; but not long enough that they get bored.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+
+mission "Remnant: Visit to Paradise 1"
+	name "Take Taely to visit <stopovers>"
+	description "Take Taely to visit Vail, then return to Caelian."
```
Do we really need to replace these? As in, is there actually a point or a solid reason for doing so? These destinations are hardcoded. There is zero possibility that they will be anything other than Vail and Caelian.

I am increasingly of the opinion that we should *not* use text replacements for cases where there is no possibility of change. This is simply adding in complexity for the sake of adding in complexity, as best I can tell.

quyykk replied:
If the destination is changed for whatever reason in the future, then we don't need to update the text every time. Also text replacements aren't that complicated.

Also useful when changing the source/destination for testing purposes I guess.

Zitchas replied:
OK, I'm opting to *not* replace them. Making it easy to identify and work with missions and identify which ones are specific locations and which ones can vary. Definitely takes precedence over the non-likely-to-every-be-useful benefit of being able to change them later.

W1zrad replied:
I'm firmly on the side of replacing these, myself. Your point is valid, but it doesn't seem reasonable to break that precedent unless there's enough support from folks generally. If you've got concerns, perhaps we could get some more community feedback or something to reach a resolution?

Zitchas replied:
I'm willing to go with the tradition for now (and the change has been applied), but I would liked to see some community consensus on having a bit of flexibility in certain places for the future.

W1zrad replied:
Sounds good!

Zitchas commented on data/remnant/remnant 2 cognizance.txt:
```diff
@@ -1618,3 +1618,13 @@ mission "Remnant: Cognizance 38"
 			`	They look around at the surrounding Remnant, and you have the sudden feeling that there is an entire level of discussion happening around you. You spot Aeolus moving through the crowd to the front, and he gestures with tones of appreciation. "Thank you for all your work on this, <first>. We have a strong appreciation for the risks you have taken alongside us, and I hope we will continue to e ...
 
+
+mission "Remnant: Cognizance 39"
```
Good catch. The mission is now invisible. That being said, the self-fail isn't needed, so I'm not doing it in order to avoid cluttering things with unnecessary fails. They're a valuable tool, but everytime there's a fail in the mission list it gives people panic attacks. In any case, it's fine if the mission takes a few days to be completed.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,149 @@
+				`	"Not right now."`
+			label vaillater
+			`	"Oh, maybe some other time, then." She looks disappointed, and settles in to eat."`
+				defer
```
I personally take the few of "Is there any consequence to delaying this? Might the player not want to do this now but do it later? If the answer to the first is no and the second yes, then it should have a defer.

Anarchist2 commented on data/remnant/remnant 2 cognizance.txt:
```diff
@@ -1618,3 +1618,14 @@ mission "Remnant: Cognizance 38"
+	landing
+	invisible
+	source
+		government "Remnant"
```
Why limit this to just Remnant space?

Zitchas replied:
Just to prevent it from lasting for too long:

To quote the documentation:
> if no destination is given that same planet is used.

So, limit the source to Remnant planets, and then there's a high chance they'll land on it again soon enough.

Anarchist2 commented on data/remnant/remnant 2 cognizance.txt:
```diff
@@ -1618,3 +1618,14 @@ mission "Remnant: Cognizance 38"
 			`	They look around at the surrounding Remnant, and you have the sudden feeling that there is an entire level of discussion happening around you. You spot Aeolus moving through the crowd to the front, and he gestures with tones of appreciation. "Thank you for all your work on this, <first>. We have a strong appreciation for the risks you have taken alongside us, and I hope we will continue to e ...
 
+
+mission "Remnant: Cognizance 39"
+	name "A breath of calm"
+	landing
+	invisible
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Cognizance 38: done"
+	on offer
+		event "remnant: cognizance calm" 5
```
Since you seem so dead-set on not creating a fail condition...
```suggestion
mission "Remnant: Cognizance 39"
	landing
	invisible
	to offer
		has "Remnant: Cognizance 38: done"
	on offer
		event "remnant: cognizance calm" 5
		fail
	on fail
		clear "Remnant: Cognizance 39: failed"
```

Anarchist2 replied:
The simplest possible solution is to just let the mission fail. Writing code to appease the miniscule proportion of people who look into their savefile but don't understand mission syntax is a fool's errand.

Zitchas replied:
Or, we can simply let it complete properly, the way missions naturally work. I've got a decent understanding of how the mission syntax works, and this appeases *my* double take every time I see a "fail" when I'm going through my save file checking something. Or more accurately, since I'm not changing any existing mission, this simply prevents adding in a new such situation.

Amazinite commented:
> Yes, this has a conversation loop.

It won't forever: #5440.

1010todd commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,151 @@
+					goto askquestion
+				`	"Not right now."`
+			label vaillater
+			`	"Oh, maybe some other time, then." She looks disappointed, and settles in to eat."`
```
```suggestion
			`	"Oh, maybe some other time, then." She looks disappointed, and settles in to eat.`
```
Extra quotation mark?

1010todd commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,151 @@
+			`	Heading back to the spaceport after a day of hoverboarding, Taely surprises you by pulling the hood open on the hovercar and spending almost an hour going over the internal systems before closing it up with a nod. "Well, that was interesting. The hovercar, I mean." She gestures at it with her commlink. "So many different manufacturers were involved in making this vehicle. Sometimes I wonder  ...
+	on complete
+		conversation
+			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with Gulls and Pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a larg ...
```
```suggestion
			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with Gulls and Pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a few minutes they deposit you on a large platform.`
```

williaji commented:
This is an interesting way to provide us with part of the Remnant backstory as well as giving us some characterization for Taely. I'm dropping my comments here as I'm not sure the best way to comment within the code without offering edits.
1) Opening: the information provided in the two options (rest/exciting) seem unbalance. In the 'taking a break' section we get a description as to why Taely needs a vacation. I took the 'exciting as always' path and it seemed sparse and perhaps stilted by comparison..
2) Vail. Was surprised that the spent so little time at the resort. It read like a day. Given the travel time there and back (~2 weeks each way), it seems like they would have more on the agenda. A few days at least. It also made me curious as to how they used the travel time. Is this enforced down time because they are out of comms range or is she working on the train and barely comes up for air? 
3) Train: Train whistle seems dated. I can't tell if it's a sound or a description of the speed of the train. I'd expect some lights or verbal: "Warning: fast train approaching" kind of thing when the energy barriers go up.
4) Two things occurred to me on the camels: 1) canal boats/tug boats are a metaphor here and 2) is there a problem with stopping one of these 'massive loads'? The mass may be free of friction, but it still has momentum. I have a vision of it running over the camel team when they try to stop. Maybe they are harnessed on the side. 
5) Stone: I apreciate your oblique hint as to the origin of Remnant sign language. Thank you for letting us put it together.
6) Wood: "They are quite different, biologically speaking..." is ambiguous to me. You might want to make a comparison between wood and the native board material explicit. "It is quite different from terran wood, biologically speaking..."
7) There is another advantage to using local materials: resiliency. Each population can be self sustaining at that level of technology, or close to it. They are essentailly choosing to live within their technological means until thay can bootstrap their industry.It rather reminds me of the Hai comment about embers and slow fires. The Remnant can't afford to waste resources, so they live sustainably.

Zitchas commented:
Alright, my apologies to reviewers for having had to take a break.

Responses to @williaji 's comments:
1. The initial "choice" now gets the same response from Taely, as they both apply. So neither option will get less of a response.
2. I intentionally pulled back from the narrative a bit in this part because I didn't want to spell out the entire visit. Let the player imagine doing what they will. As for the time frame, time progresses one day between landing and take-off, so that's the time they have on planet. Taely's not the sort to waste too much time, either. Not narrating the whole stay also helps reduce the text-wall a bit.
3. The whistle is intended to be from the motion, not a literal train whistle. I have adjusted this to be "whooshes" past.
4. That's a good point. Especially for heavy loads, they most likely would have the camels pulling from the side, maybe even teams following form the rear to provide braking force. That being said, it is also possible the repulsors might have brakes of some kind.
5. You are welcome. I'm glad you picked up on that. (and appreciate it)
6. That's a good point. I have adjusted that comparison to be more explicit about comparing them to Earth trees.
7. Yes, exactly. The Remnant had no recourse for additional materials. In some regards, this is hinting at the Korath dilemna: They have all this ultra-high-tech knowledge; but have had their resource base reduced to a tiny fraction of what it was. For the Remnant, of course, they were at human 25th century level tech, and forced in some ways to dip back to tech that was commonplace in the 1500-1700s. But it was the same sort of "advanced knowledge, inadequate resources" situation. This is also intended to be a very subtle hint at the potential of the Korath to recover and exceed.


The preamble before the acceptance choice now mentions hoverboarding; so the player's option to comment on that isn't out of nowhere. It also provides the player with a bit more context if they don't remember that Veil is the hoverboarding world.

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,146 @@
+			`As you approach, your navigation system picks up the beacons designating the approach path, and before long you are automatically guided to a landing pad on the outskirts of the main spaceport.`
+				goto vaillanding
+			label science
+			`As you approach Vail, your sensors detect parts of the weather control network tasked with increasing the amount of snowfall on the planet. Your navigation system picks up the beacons designating the approach path, and before too long you are automatically guided down to a landing pad on the outskirts of the main spaceport.`
```
```suggestion
			`As you approach Vail, the specialized sensors from your Remnant research laboratory detect parts of the weather control network tasked with increasing the amount of snowfall on the planet. Your navigation system picks up the beacons designating the approach path, and before too long you are automatically guided down to a landing pad on the outskirts of the main spaceport.`
```
Or something like this? Could be nice to note that the reason we're getting this info is the lab.

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,146 @@
+	on complete
+		conversation
+			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with Gulls and Pelicans shifting larger quantities of products. Disembaring from the <ship>, she leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a few minutes ...
+			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier," she says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature, but they can still hurt if you run into them." She gestures to easily recognizabl ...
```
All I can think about is a scenario where firstlast walks straight into one of these and smacks right into it, full glass-door style :P

Zitchas replied:
Maybe I will have that happen in a different side mission, one where their Remnant escort is distracted or something.

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,146 @@
+		conversation
+			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with Gulls and Pelicans shifting larger quantities of products. Disembaring from the <ship>, she leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a few minutes ...
+			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier," she says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature, but they can still hurt if you run into them." She gestures to easily recognizabl ...
+			`	Moments later, you see why the barriers are in place as a large train whooshes into the station at high speed before quickly coming to a stop. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering and exiting the train. Within moments the doors close and the train starts moving again as y ...
```
```suggestion
			`	Moments later, you see why the barriers are in place as a large train whooshes into the station at high speed before quickly coming to a stop. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering and exiting the train. Within moments the doors close, and the train starts moving again as you are pressed back into your seat. You see a few lights flicker past outside the window, but soon there is nothing outside but a monotonous dark blur.`
```

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,146 @@
+				`	"Thanks for the explanation."`
+					goto ourdestination
+				`	"What is a coefficient?"`
+			`	She pulls up a hologram with c=11'DE7'84Am/s on it. "Here, I will switch to base 10 to make it easier for you to understand." As she speaks, the number 299'792'458m/s appears on the screen, and then changes to 2.997'924'58x10^8m/s. "When we write out a number such as the speed of light, it is typical to break it into parts. This is generally referred to as scientific notation. This is done t ...
```
Is the use of apostrophes to separate digits a hexadecimal thing, Remnant thing, or neither? Some of your notation seems odd here - I can't understand it, and it isn't explained. For context, American English would use commas to separate every three whole digits and periods to separate between whole numbers and decimals. For instance, one thousand five hundred and four hundredths would be "1,500.04" (I promise I'm not trying to be patronizing if it comes off this way, just trying to understand what the intent here is and make sure what I'm thinking is clear).

Zitchas replied:
Let's call it a Remnant thing. It's actually a personal habit, because I've had to deal with a lot of places where the , and . and things look too similar and generate confusion. Given that the Remnant are kind of a high-pressure "need to understand stuff at a glance" types, it makes sense that they'd adopt a digit grouping separator that won't generate confusion and erroneous readings.

W1zrad replied:
Got it. I don't know if it's just me, but my concern here is that your numbering system is abnormal enough that your explanation that uses it loses its usefulness as an explanation by using something that's not well-known (at least in my experience). For instance, I have no idea what number 2.997'924'58 actually is.

Zitchas replied:
I see your point. I think it probably works to the left of the decimal, but it gets really hard to read in the scientific notation. I think that is a case of one or the other, not both.

Zitchas replied:
Alright. This has been changed. Added a bunch of spaces to spread it out a bit; and took all the apostrophes out of the scientific notation.

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,146 @@
+			choice
+				`	"I do not know what to make of this."`
+				`	"This is a spaceport?"`
+			`	She nods, and guides you amongst the stream of camel-drawn hovercarts heading for the gate. "This is one of our earliest spaceports, before we learned new techniques better suited to our new home. Our founding ships brought as much as they could, of course, but as it turns out, the earliest phases of expansion look much the same whether one is looking at historic pre-flight eras, or modern s ...
```
```suggestion
			`	She nods, and guides you amongst the stream of camel-drawn hovercarts heading for the gate. "This is one of our earliest spaceports, before we learned new techniques better suited to our new home. Our founding ships brought as much as they could, of course, but as it turns out, the earliest phases of expansion look much the same, whether one is looking at historic pre-flight eras or modern settlement efforts."`
```

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,146 @@
+					goto stonewalls
+				`	"Are those wind turbines?"`
+					goto windturbine
+				`	"All these wood products seems surprising."`
```
```suggestion
				`	"All these wood products seem surprising."`
```

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,146 @@
+				goto questionloop
+			label camelcart
+			scene "scene/remnant remote spaceport"
+			`	"Is that so? The engineering advantages are quite simple: With the mechanics of the cart focused entirely on supporting the load and the camel providing horizontal motion, the cart can support much larger masses than it could if it had to split the forces." She looks wistful. "In truth, I am somewhat envious of the industrial capacity the ancestral worlds contain - We have accomplished much, ...
```
```suggestion
			`	"Is that so? The engineering advantages are quite simple: with the mechanics of the cart focused entirely on supporting the load and the camel providing horizontal motion, the cart can support much larger masses than it could if it had to split the forces." She looks wistful. "In truth, I am somewhat envious of the industrial capacity the ancestral worlds contain - we have accomplished much, but we have had to make choices, and one of those was the decision as to where to spend our industrial capacity. We chose, obviously, to focus on spacecraft and heavy industrial applications. We have no cars, no personal vehicles at all, really. Even these hovercarts are basically the repulsor systems from our spacecraft that have been repurposed to carry massive loads."`
```

W1zrad commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -0,0 +1,146 @@
+				goto questionloop
+			label windturbine
+			scene "scene/remnant remote spaceport"
+			`	"They are indeed wind turbines." She steps through the gate to a spot with a better view. You can see that while the ones on the wall are are only a couple of meters tall, the ones on top of the towers are significantly bigger. "Our ancestors rightly guessed that anywhere that had sufficient atmosphere to dispense with pressure suits would also have wind. The vertical style also has the adva ...
```
Two concerns:
1. `The vertical style also has the advantage that it handles high wind speeds and rapid variation in direction much better than some of the shapes that were more traditional at the time of the exodus.`
Are we missing a word here? Can't understand the `direction` bit so I thought that might be the case

2. `A good precaution, which has enabled these to survive the harsh windstorms that can sweep across the continent.`
This is an incomplete sentence. I'd suggest a change, but I just need to understand what the first one's saying before I come up with something.

Zitchas replied:
1. "rapid variation in direction" or in other words, dealing with winds that change directions really frequently and abruptly.

W1zrad replied:
```suggestion
			`	"They are indeed wind turbines." She steps through the gate to a spot with a better view. You can see that while the ones on the wall are are only a couple of meters tall, the ones on top of the towers are significantly bigger. "Our ancestors rightly guessed that anywhere that had sufficient atmosphere to dispense with pressure suits would also have wind. The vertical style also has the advantage that it handles high wind speeds and rapid variation in direction much better than some of the shapes that were more traditional at the time of the exodus. It has enabled these to survive the harsh windstorms that can sweep across the continent. While they have been repaired and improved over the centuries, these wind turbines still provide sufficient power for much of the agricultural uses out here, so we let them continue. They are also useful to remind us from where we have come."`
```
OR
```suggestion
			`	"They are indeed wind turbines." She steps through the gate to a spot with a better view. You can see that while the ones on the wall are are only a couple of meters tall, the ones on top of the towers are significantly bigger. "Our ancestors rightly guessed that anywhere that had sufficient atmosphere to dispense with pressure suits would also have wind. The vertical style also has the advantage that it handles high wind speeds and rapid variation in direction much better than some of the shapes that were more traditional at the time of the exodus. It seems to be a good precaution, as it has enabled these to survive the harsh windstorms that can sweep across the continent. While they have been repaired and improved over the centuries, these wind turbines still provide sufficient power for much of the agricultural uses out here, so we let them continue. They are also useful to remind us from where we have come."`
```
Thanks! Take your pick.

W1zrad reviewed (changes requested):
Good stuff! Fun lore and worldbuilding that feels natural; always a plus. One thing - you might want to ask a coder about displaying the Remnant spaceport image after every choice in your conversation loop at the end. I don't know enough about coding and computers to know for sure if it's an issue, but it seems like that could maybe be memory-heavy or something?
